FAQ.md

```diff
@@ -141,6 +143,26 @@ cache:
 
 Remove the cache section from your buildspec.
 
+## Console
+
+### Amplify app not showing build status and last updated time in the Amplify Console all apps view
+
+This particular behavior can occur if your Amplify app does not have a `PRODUCTION` [stage](https://docs.aws.amazon.com/amplify/latest/APIReference/API_CreateBranch.html#amplify-CreateBranch-request-stage) branch associated to it.
+
+To list the apps in the Amplify Console UI, we use the [ListApps API](https://docs.aws.amazon.com/amplify/latest/APIReference/API_ListApps.html) and it uses the `status` attribute of the production branch to display the build status and last deploy time: https://docs.aws.amazon.com/amplify/latest/APIReference/API_ProductionBranch.html#amplify-Type-ProductionBranch-status
+
+To associate a `PRODUCTION` stage to your app’s branch, you will need to complete the following steps:
+
+- Navigate to the Amplify Console and select the Amplify app
+- Under `App settings` navigate to `General`
+- Click the `Edit` button which is located on the top right corner of the console
+- Under the `Repository settings` section, navigate to the `Production environment` field
+- From the `Production environment` field dropdown, select the desired branch name
```

Comment:
Nit: this shows as `Production branch` for me instead of `Production environment`
![Screenshot 2023-06-08 at 6 03 12 PM](https://github.com/aws-amplify/amplify-hosting/assets/87148442/fa46db83-74a7-4a25-bcd4-21f009629635)

Reply:
Good call out!

`Production environment` -> Manually deployed apps
`Production branch` -> CI/CD deployed apps